src/tests/ftest/util/mpio_utils.py

```diff
@@ -137,13 +138,22 @@ def run_llnl_mpi4py(self, basepath, hostfile, pool_uuid, test_repo,
                                                       "/test_io_daos.py"):
             test_cmd = "mpiexec -n {} --hostfile {} python test_io_daos.py"\
                            .format(client_processes, hostfile)
+        elif test_name == "hdf5" and (os.path.isfile(test_repo + "/testphdf5")
+                                      and os.path.isfile(test_repo + "/t_shapesame")):
```

Comment:
(pylint-line-too-long) Line too long (86/80)

Reply:
should be able to rework this to get line under 80

Reply:
This message was in older patch, was taken care-off in later patches.